CMakeLists.txt

```diff
@@ -303,10 +312,8 @@ add_custom_command(
 add_custom_target(accp-jar-source DEPENDS ${ACCP_JAR_SOURCE})
 
 if(ENABLE_NATIVE_TEST_HOOKS)
-  add_executable(test_keyutils EXCLUDE_FROM_ALL
-        csrc/test_keyutils.cpp
-    )
-    link_with_openssl(test_keyutils)
```

Comment:
why don't we need to link the test utils with AWS-LC anymore?

Reply:
AWS-LC is statically linked into libaccp. AWS-LC has made some [changes](https://github.com/aws/aws-lc/pull/1332) to how the FIPS integrity checks works, and as a result, when AWS-LC is linked statically twice, on some platforms, like aarch64, it fails.